pycroft/helpers/printing/__init__.py

```diff
@@ -84,39 +84,40 @@ def generate_user_sheet(user, user_id, plain_password, generation_purpose=''):
                   ('VALIGN', (0, 0), (-1, -1), 'MIDDLE'),
               ])
     story.append(t)
-    ######################
 
     story.append(HRFlowable(width="100%",
                             thickness=1,
                             color=black,
                             spaceBefore=0.0 * cm,
-                            spaceAfter=0.8 * cm))
+                            spaceAfter=0.5 * cm))
 
-    story.append(
-        Paragraph('Welcome as a member of the AG DSN, {}!'
+    welcome = Paragraph('''Welcome as a member of the AG DSN, {}!
+We are proud to announce that your network access has been activated. If you encounter any problems, drop us a mail or visit us during our office hours. You can find contact information below on this page.'''
                   .format(user.name),
-                  style['BodyText']))
-
-    story.append(
-        Paragraph('We are proud to announce that your network access has been '
-                  'activated. If you encounter any problems, drop us a mail or '
-                  'visit us during our office hours. You can find contact '
-                  'information below on this page.',
-                  style['BodyText']))
+                  style['BodyText'])
+
+    return_notice = Paragraph(
+        '''<font size="9pt">Nicht nachsenden!</font>''',
+        style['Normal'])
+    sender = Paragraph(
+        '''<font size="9pt">AG DSN • BuS • Wundtstraße 5 • 01217 Dresden</font>''',
```

Comment:
Please refrain from abbreviations like `BuS` whose meaning is only known to active members of the AG DSN or related persons.